src/components/SettingsDialog/ProfileSettings/ProfileSettings.tsx

```diff
@@ -1,27 +1,31 @@
 import classNames from "classnames";
 import {useTranslation} from "react-i18next";
-import {useState} from "react";
+import {useContext, useState} from "react";
 import store, {useAppSelector} from "store";
 import {Actions} from "store/action";
 import "./ProfileSettings.scss";
 import {useDispatch} from "react-redux";
 import {ReactComponent as InfoIcon} from "assets/icon-info.svg";
 import {Toggle} from "components/Toggle";
+import {ApplicationState} from "types";
 import {AvatarSettings} from "../Components/AvatarSettings";
 import {SettingsInput} from "../Components/SettingsInput";
 import {SettingsButton} from "../Components/SettingsButton";
+import {Footer} from "../Components/Footer";
+import {SettingsContext} from "../settingsContext";
 
 export const ProfileSettings = () => {
+  const {settings} = useContext(SettingsContext);
   const {t} = useTranslation();
   const dispatch = useDispatch();
-
-  const state = useAppSelector((applicationState) => ({
+  const state = useAppSelector((applicationState: ApplicationState) => ({
```

Comment:
As far as I know the type of `applicationState` is already inferred so you don't need to type it manually